src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -436,6 +447,35 @@ def to_ase_atoms(self, logger: 'BoundLogger') -> Optional[ase.Atoms]:
 
         return ase_atoms
 
+    def from_ase_atoms(self, ase_atoms: ase.Atoms, logger: 'BoundLogger') -> None:
+        """
+        Parses the information from an ASE Atoms object to the `AtomicCell` section.
+
+        Args:
+            ase_atoms (ase.Atoms): The ASE Atoms object to parse.
+            logger (BoundLogger): The logger to log messages.
+        """
+        # `AtomsState[*].chemical_symbol`
+        chemical_symbols = ase_atoms.get_chemical_symbols()
+        for symbol in chemical_symbols:
```

Comment:
if the return type `ase_atoms.get_chemical_symbols` is `list[str]`, then you can contract these lines into 1: no need for single-use varirable assignment